res/controllers/Numark-Mixtrack-Pro-scripts.js

```diff
@@ -187,6 +207,21 @@ NumarkMixTrackPro.init = function(id) {
     );
 };
 
+
+NumarkMixTrackPro.pitchFader = function(
+    channel,
+    control,
+    value,
+    status,
+    group
+) {
```

Comment:
unusual formatting, did eslint propose this?

Reply:
prettier is only supposed to run on yaml files. Did you run prettier manually on this or did pre-commit do that?